bespin_web/tasks/loaddata.yml

```diff
@@ -41,3 +39,8 @@
     - "creategroupmanagerconnection ${GROUP_MANAGER_ACCOUNT_ID} ${GROUP_MANAGER_PASSWORD}"
```

Comment:
Out of curiosity, why use environment `${ }` variables rather than `{{ }}` ansible variables here?

Reply:
I just copied the contents of the existing scripts into here. I could probably remove the env variables all together and just use ansible variables.